README.md

```diff
@@ -27,122 +25,76 @@ iOS targets iOS 10.0+. Aries Bifold can only be run on physical devices as of ri
 
-Installation instructions are documented [here](https://reactnative.dev/docs/environment-setup).
+Aries Bifold also utilizes the [Aries Toolbox](https://github.com/hyperledger/aries-toolbox) & [aries-acapy-plugin-toolbox](https://github.com/hyperledger/aries-acapy-plugin-toolbox) for issuer and verifier roles.
 
-Troubleshooting guides can be found in the [docs/installation](./docs/INSTALLATION.md) directory.
+## Install
 
-[Cocoa Pods](https://cocoapods.org/) (iOS specific)
+You can watch a recording of setting up and running the mobile wallet and receiving a credential using the ACA-Py demo. Watch the video [here](https://youtu.be/AomoHvw4lgc) (thanks [@xtrycatchx](https://github.com/xtrycatchx)).
+
+1. React Native Setup:
+    * React Native installation instructions are documented [here](https://reactnative.dev/docs/environment-setup).
+    * (iOS) Install [Cocoa Pods](https://cocoapods.org/)
+2. Clone the Bifold repo and install its dependencies:
+    ```sh
+    git clone https://github.com/hyperledger/aries-mobile-agent-react-native
+    cd aries-mobile-agent-react-native
+    npm install
+    ```
+3. (iOS) iOS specific configuration:
+    * Install iOS Pods:
+        ```sh
+        cd ios
+        pod install
+        ```
+    * In the /ios directory, open the project workspace file in Xcode.
+    Once the project is open, navigate to the project's Signing & Capabilities tab and apply your personal Apple Developer Account or your organization's team to target AriesBifold and target AriesBifoldTests.
+    * Adjust the bundle identifier if needed.
+3. Configure Bifold:
+    In the root directory add an `.env` file containing:
+    ```
+    MEDIATOR_URL=http://mediator3.test.indiciotech.io:3000?c_i=eyJAdHlwZSI6ICJkaWQ6c292OkJ6Q2JzTlloTXJqSGlxWkRUVUFTSGc7c3BlYy9jb25uZWN0aW9ucy8xLjAvaW52aXRhdGlvbiIsICJAaWQiOiAiYjE5YTM2ZjctZjhiZi00Mjg2LTg4ZjktODM4ZTIyZDI0ZjQxIiwgInJlY2lwaWVudEtleXMiOiBbIkU5VlhKY1pzaGlYcXFMRXd6R3RtUEpCUnBtMjl4dmJMYVpuWktTU0ZOdkE2Il0sICJzZXJ2aWNlRW5kcG9pbnQiOiAiaHR0cDovL21lZGlhdG9yMy50ZXN0LmluZGljaW90ZWNoLmlvOjMwMDAiLCAibGFiZWwiOiAiSW5kaWNpbyBQdWJsaWMgTWVkaWF0b3IifQ==
+    GENESIS_URL=https://raw.githubusercontent.com/Indicio-tech/indicio-network/main/genesis_files/pool_transactions_testnet_genesis
+    ```
+    Note - To run your own mediator or use a different network, go [here for advanced configuration](#Advanced-Configuration).
+4. Run Bifold:
+    * Launch the metro bundler: 
+        ```sh
+        npm run start
+        ```
+    * Open a second terminal and run:
+        * (Android)
+            ```sh
+            npm run android
+            ```
+        * (iOS)
+            ```sh
+            npm run ios
+            ```
+    * (iOS) Via Xcode:
+        Choose your physical iOS device as the destination. Click the "Play" button to Build and Run.
```

Comment:
Maybe it's an idea to create an additional section/quote (something that stands out) that states Bifold currently doesn't work in the iOS simulator and thus requires a physical iOS device? That might avoid some confusion for devs getting started with Bifold.